### DISCUSSION TITLE
InfoRepo Discovery

### DISCUSSION THREAD

**Discussion by heandw2024** [jmmorato/openddsharp#240] [Q&A]
I have already completed the 'HelloWorld' example using RTPS Discovery. Now I want to implement InfoRepo Discovery. Both the publisher and subscriber are on the same computer (assuming the computer is not connected to the internet), and I have tried a few times but failed. Could you help me with this?

**Comment by jmmorato** [comment]
You can see an example on how to configure InfoRepo in the unit test project: https://github.com/jmmorato/openddsharp/blob/339a136a571b1d440783a4a5cd8f5fb3a1a57b4b/Tests/OpenDDSharp.UnitTest/AssemblyInitializer.cs#L75C12-L83C49

**Reply by jmmorato** [reply]
After that, also configure the transport in your participant, something like: https://github.com/jmmorato/openddsharp/blob/339a136a571b1d440783a4a5cd8f5fb3a1a57b4b/Tests/OpenDDSharp.UnitTest/TopicBuiltinTopicDataDataReaderTest.cs#L62C11-L64C51

**Reply by heandw2024** [reply]
I might have overlooked an issue; I should download the OpenDDS source code and compile InfoRepo.exe.
